source/clear-linux/reference/image-types.rst

```diff
@@ -51,31 +48,46 @@ Table 2 lists the currently available images that are platform specific.
    * - Image Type
      - Description
 
+   * - aws.img
+     - Image suitable for use with Amazon\* AWS\*.
+
    * - azure.vhd
-     - Virtual Hard Disk for use on Microsoft\* Azure\* cloud platform
+     - Virtual Hard Disk for use on Microsoft\* Azure\* cloud platform.
 
    * - azure-docker.vhd
-     - Virtual Hard Disk for use on Microsoft Azure cloud platform with Docker\* pre-installed
+     - Virtual Hard Disk for use on Microsoft Azure cloud platform with Docker\* pre-installed.
 
    * - azure-machine-learning.vhd
-     - Virtual Hard Disk for use on Microsoft Azure cloud platform with the `machine-learning-basic` bundle installed
+     - Virtual Hard Disk for use on Microsoft Azure cloud platform with the `machine-learning-basic` bundle installed.
 
    * - cloud.img
```

Comment:
@gtkramer , Didn't cloud get replaced by cloudguest or do both now exist?

Reply:
@tpleavitt. As of this release there is still a cloud version. That’s why I left it.

Reply:
Let's hear from @gtkramer first.  There is an email thread on dev-public about the images and discusses cloud.img and cloudguest.img.  @bryteise , can you take a look at the proposed change to the image description on the docs site.

Reply:
@tpleavitt @intelkevinputnam the cloud image is being removed and will be replaced with cloudguest.  We are in a deprecation period with removal scheduled for next Tuesday.  This rename of cloud to cloudguest has already occurred on https://clearlinux.org/downloads.